### DISCUSSION TITLE
API key visible in reporting

### DISCUSSION THREAD

**Discussion by HartleSeb** [thunderclient/thunder-client-support#1372] [Q&A]
Hello community!

Is there a way to secure your data like tokens and api keys in the reporting? We want to publish our reports but then we noticed that the html report gives the full request header with our personal api keys. Is there a way to remove that part automatically during report generation or just returning the varibale name in that case?

Someone having the same issues?

Kind regards
Sebastian

**Comment by rangav** [comment]
Hi @HartleSeb 

Thanks for the feedback. Will review and provide hide options when running in the CLI.

**Reply by rangav** [reply]
Can you please create a new feature request

**Comment by rangav** [comment]
This feature is implemented in CLI in `v1.7.1`

How to use 
- `tc --col "name" --report "html" --remove "Auth,API-KEY"`
- for more details `tc -h`

https://github.com/rangav/thunder-client-support/releases/tag/v2.13.3

Please let me know your feedback.
